widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -274,8 +274,12 @@ public static void getVisitorInfo(Consumer<GliaVisitorInfo> visitorCallback, Con
 
     /**
      * Allows configuring custom response cards based on metadata.
+     * <p>
+     * Glia SDK uses {@link WebViewCardAdapter} by default.
```

Comment:
are the publicly facing endpoints synced with iOS?

Reply:
We have an adapter, they have a renderer. But the default logic between the platforms was synchronized.